app/view_models/sign_up_completions_show.rb

```diff
@@ -19,11 +19,13 @@ def initialize(ial2_requested:, decorated_session:, current_user:, handoff:, ial
     [[:birthdate], :birthdate],
     [[:social_security_number], :social_security_number],
     [[:x509_subject], :x509_subject],
+    [[:verified_at], :verified_at],
```

Comment:
@jmhooper @stevegsa Question: there's now much low-level test coverage of this form, so it's not easy to add checks that these work. It's tested through the acceptance test that checks for the copy on the final page, is that enough?

Reply:
Yeah, for completions show the important thing is that it is visible to the user, so that is probs okay.

If we want to go the extra mile we could throw something in there to confirm that we added the scope to the `verified_attributes` column.

Reply:
Sounds good! Added in c109c569146db77f459cf40804f9019692121f71